tests/qm-kvm-test/kvm-basic/check_container_kvm_run_in_qm.sh

```diff
@@ -32,6 +32,37 @@ copy_kvm_configuration() {
     info_message "PASS: Copy configuration finished."
 }
 
+wait_for_service_active() {
+    local container="$1"
+    local service="$2"
+    local timeout="${3:-30}"  # Default timeout is 30 seconds
+
+    if [ -n "${container}" ];then
+        info_message "Waiting for service '$service' in container '$container' to become active..."
+        while ! podman exec "$container" systemctl is-active --quiet "$service"; do
```

Comment:
can we have a function to avoid duplicate code? It seems a dup code on the if/else